new/LIT6351MEMiracle.xml

```diff
@@ -0,0 +1,106 @@
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2021-01-20">Created entity</change>     
+            <change who="MV" when="2021-01-21">Added bibliography, fixed title</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Turaiev1905Eustathii"/>
+                        <citedRange unit="page"><!-- insert page range from Turaiev's Monumenta Aethiopiae Hagiologica: Vita et Miracula Eustathii, pp. 134-177 --> (2nd miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+                <listBibl type="translation">
+                    <bibl>
+                        <ptr target="bm:Turaev1906Ewostatewos"/>
+                        <citedRange unit="page">72-73 (2nd miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:GrebTiss1935Codices"/>
+                        <citedRange unit="page">196</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:EMML5"/>
```

Comment:
you could add here that is the first miracle in this ms (1st miarcle)
Or is it redundant for the catalogue?

Reply:
I have added all the correspondences with the miracles in EMML 1636 in this way, e.g:
  `<citedRange unit="item">EMML 1636 2.6 (6th miracle)</citedRange>`
like you did for the miracles of George. It did not take much.